src/sentry/incidents/subscription_processor.py

```diff
@@ -756,6 +742,43 @@ def get_anomaly_data_from_seer(self, aggregation_value: float | None):
             )
             return None
 
+        try:
+            results: DetectAnomaliesResponse = json.loads(decoded_data)
+        except JSONDecodeError:
+            logger.exception(
+                "Failed to parse Seer anomaly detection response",
+                extra={
+                    "ad_config": anomaly_detection_config,
+                    "context": context,
+                    "response_data": decoded_data,
+                    "reponse_code": response.status,
+                },
+            )
+            return None
+
+        if not results.get("success"):
```

Comment:
So Seer can return 200 but still have a failure response?

Reply:
I don't really expect it to, just trying to cover all the bases